diff --git a/node_modules/freesound/freesound.js b/node_modules/freesound/freesound.js
index 879d67a..25a1ca1 100644
--- a/node_modules/freesound/freesound.js
+++ b/node_modules/freesound/freesound.js
@@ -57,46 +57,24 @@
                 uri = uri +"?"+ paramStr;
             }
             
-            if (typeof module !== 'undefined'){ // node.js
-                var http = require("http");
-                var options = {
-                    host: host,
-                    path: uri.substring(uri.indexOf("/",8),uri.length), // first '/' after 'http://'
-                    port: '80',
-                    method: method,
-                    headers: {'Authorization': authHeader}
-                };
-                var req = http.request(options,function(res){
-                    res.setEncoding('utf8');            
-                    res.on('data', function (data){ 
-                        if([200,201,202].indexOf(res.statusCode)>=0)
-                            success(wrapper?wrapper(data):data);
-                        else   
-                            error(data);
-                    });                    
-                });                
-                req.on('error', error).end();
-            }
-            else{ // browser
-                var xhr;
-                try {xhr = new XMLHttpRequest();}
-                catch (e) {xhr = new ActiveXObject('Microsoft.XMLHTTP');}
-
-                xhr.onreadystatechange = function(){
-                    if (xhr.readyState === 4 && [200,201,202].indexOf(xhr.status)>=0){
-                        var data = eval("(" + xhr.responseText + ")");
-                        if(success) success(wrapper?wrapper(data):data);
-                    }
-                    else if (xhr.readyState === 4 && xhr.status !== 200){
-                        if(error) error(xhr.statusText);
-                    }
-                };
-                xhr.open(method, uri);
-                xhr.setRequestHeader('Authorization',authHeader);
-                if(content_type!==undefined)
-                    xhr.setRequestHeader('Content-Type',content_type);
-                xhr.send(data);
-            }
+            var xhr;
+            try {xhr = new XMLHttpRequest();}
+            catch (e) {xhr = new ActiveXObject('Microsoft.XMLHTTP');}
+
+            xhr.onreadystatechange = function(){
+                if (xhr.readyState === 4 && [200,201,202].indexOf(xhr.status)>=0){
+                    var data = eval("(" + xhr.responseText + ")");
+                    if(success) success(wrapper?wrapper(data):data);
+                }
+                else if (xhr.readyState === 4 && xhr.status !== 200){
+                    if(error) error(xhr.statusText);
+                }
+            };
+            xhr.open(method, uri);
+            xhr.setRequestHeader('Authorization',authHeader);
+            if(content_type!==undefined)
+                xhr.setRequestHeader('Content-Type',content_type);
+            xhr.send(data);
     };
     var checkOauth = function(){
         if(authHeader.indexOf("Bearer")==-1)
